### PR TITLE
cleanup fast-usdc package

### PR DESCRIPTION
**Pull request by turadg**
refs: #10328 

## Description
Following up on package creation. See changes in commit list. Principally renaming the package to have the `@agoric/` prefix.

### Security Considerations
`@agoric/` could reduce NPM squatting risk

### Scaling Considerations
n/a

### Documentation Considerations
none

### Testing Considerations
one

### Upgrade Considerations
not deployable

### CODE REVIEW

**Review by michaelfig — APPROVED** [review]
LGTM!  Thanks for this attention to detail.

**Comment by michaelfig** [comment]
Looks like [CI is failing to build the PR](https://github.com/Agoric/agoric-sdk/actions/runs/11583506015/job/32248799748?pr=10364#step:5:14):

```
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   packages/agoric-cli/src/sdk-package-names.js
+ echo 'Unexpected dirty git status in Agoric SDK path'
```

Can you try running a `yarn build` at the top of your `agoric-sdk` checkout, and committing and pushing the result to this PR?  That seems to produce a different `sdk-package-names.js` than the one in the current PR.

File: packages/fast-usdc/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "fast-usdc",
```
Comment (michaelfig) [inline packages/fast-usdc/package.json]: Oh.  Didn't you mean to change this?
```suggestion
  "name": "@agoric/fast-usdc",
```

Comment (turadg) [reply packages/fast-usdc/package.json]: lol, I did mean to! https://github.com/Agoric/agoric-sdk/pull/10364/commits/139d42b5629118ba57205d5d0d72ed99b9b3aad9

**Pull request merged**